### DISCUSSION TITLE
Can't cycle through suggestions

### DISCUSSION THREAD

**Discussion by pdavis68** [community/community#6667] [Copilot Conversations]
I'm running under Windows. According to the docs, to cycle through alternative suggestions:

- On Windows or Linux, press Alt+] (or Alt+[).

But nothing happens when I do either key combo. Is this feature not in yet?

So far I'm really impressed with some of the stuff it comes up with. It's a little rough around the edges, but looks like it's got the potential to be an extraordinary time saver and a real game changer for developers.  Anything that does away with tedious tasks, gives developers more time to think about the deeper issues.

**Comment by pdavis68** [comment]
As I've created other classes, this feature seems to work, but in some cases it doesn't. I seem to be able to reliably reproduce it with this C# class:

```
    public class Stock
    {
        public string Symbol { get; set; }

    }
```

If I then go to the line after "Symbol", it will suggest:

`public decimal Price { get; set; }`

If I don't want "Price", it will offer me no other alternatives. And perhaps there are no other alternatives for it at this point. That would be nice to know. But it doesn't do anything to use Alt+']'

Now, if I hit tab and accept "Price"

The next thing offered up is:
`public decimal Change { get; set; }`

And I can get an alternative to this.

So perhaps the issue is that sometimes there are no alternatives. It would be nice if there were some sort of indicator for this condition, if that's the case.

**Comment by hsenag** [comment]
Hi, and thanks for the feedback! You're right that there aren't always alternatives available. If you hover over the suggestion, you should be able to see if another suggestion is available or not - for example in the case below the options are greyed out:
<img width="425" alt="Screenshot 2021-10-23 at 23 54 37" src="https://user-images.githubusercontent.com/451141/138573910-014dfa40-bb17-4fda-b396-99cbb3798f02.png">
